app/uk/gov/hmrc/gform/graph/Recalculation.scala

```diff
@@ -27,9 +27,10 @@ import scalax.collection.GraphEdge._
 import shapeless.syntax.typeable._
```

Comment:
all changes in the class is just refactoring, to keep the core method small and simple to read